test/algconvergence/ode_convergence_tests.jl

```diff
@@ -33,6 +33,9 @@ testTol = 0.2
   sim3 = test_convergence(dts2,prob,KuttaPRK2p5(threading=false))
   @test sim3.𝒪est[:l∞] ≈ 5 atol=testTol
 
+  sim3 = test_convergence(dts2,prob,RKO65())
```

Comment:
what does `plot(sim3)` look like?

Reply:
Foe the 2D inlace: 

![prob_ode_2Dlinear](https://user-images.githubusercontent.com/42201748/73768479-5c195380-4779-11ea-9036-875d547b5c5f.png)

Reply:
For the 1D out of place

![prob_ode_linear](https://user-images.githubusercontent.com/42201748/73768550-810dc680-4779-11ea-936a-31b483389881.png)